aesara/scan/scan_perform.pyx

```diff
@@ -205,8 +205,6 @@ def perform(
         raise IndexError(
             "Scan was asked to run for negative number of step %d" %
             n_steps)
-    elif n_steps == 0:
-        raise NotImplementedError("n_steps == 0")
```

Comment:
You might need to return something here, because who knows what the code below is doing when this is `0`.

Reply:
In a previous version I returned here, but it does not change anything in practice. Probably safer and more explicit so I will do that.